rtgui/guiutils.h

```diff
@@ -430,6 +436,22 @@ class MyImageMenuItem : public Gtk::MenuItem
     const Gtk::Label* getLabel () const;
 };
 
+class MyProgressBar : public Gtk::ProgressBar
```

Comment:
Things named `utility` or `miscellaneous` tend to be dumping grounds for all kinds of code, hence I'd suggest to use `widgets`. (And even if the `My*` do only small extensions to their base classes, I think it is important to think of them as fully-fledged widgets that only happen to inherit most of the interface and implementation as even small extension can significantly change their behaviour.)